src/formatDate.js

```diff
@@ -51,6 +51,32 @@
 
 function formatDate(date, fromFormat, toFormat) {
   // write code here
+  const oldSeparator = fromFormat[3];
```

Comment:
It is better to avoid exact numbers because it's not always obvious why do we use 3 here)
`fromFormat[fromFormat.length - 1] `looks better